src/state/categories/types.js

```diff
@@ -0,0 +1,6 @@
+export const SET_CATEGORIES = 'categories/set-categories';
+export const SET_CATEGORY_SETTING = 'categories/set-category-setting';
+export const TOGGLE_CATEGORY_EXPANDED = 'categories/toggle-category-expanded';
+export const SET_CATEGORY_LOADING = 'categories/set-category-loading';
+export const SET_CATEGORY_LOADED = 'categories/set-category-loaded';
+export default SET_CATEGORIES;
```

Comment:
Why SET_CATEGORIES is also exported as default? Please remove default export, we should only use named exports for categories